getTags.js

```diff
@@ -0,0 +1,35 @@
+const extractTagsArgFromCli = cliArg => {
+  // e.g. cliArg = `cypress features/ --tags "~@ignore and @slow"`
+  const tagsRegex = new RegExp('--tags (["a-zA-Z·@~ -](?!--))+'); // eslint-disable-line no-control-regex
+  const tagsPassed = cliArg.match(tagsRegex); // e.g. `--tags "~@ignore and @slow"`
+  return tagsPassed
+    ? tagsPassed[0].replace(/"/g, "") // e.g. `"~@ignore and @slow"` -> `~@ignore and @slow`
+    : "";
+};
+
+const tagsStringToArray = tagsString =>
+  tagsString
+    .split(" ") // e.g. ["~@ignore", "and", "@slow"]
+    .filter(elm => !!elm.match(/(?:@|~).+$/g)); // e.g. ["~@ignore", "@slow"]
```

Comment:
I'm not sure I understand - does it mean we can't really use words like and/or etc here? I think that's a good starting point, but then we should probably change the documentation that suggests that we can use the whole complex syntax for deciding on tags. 
I hope I'm missing something :)

Reply:
Tags are either not supplied (so we run everything) or supplied (e.g. `@ci`, so we run only those tagged as `@ci`), or negatively supplied (e.g. `~@ignore`, so we run everything EXCEPT `@ignore` tags).

It starts getting trickier when you combine them. The spec allows something like `@ci and @fast`, so we will only run things tagged as both `@ci` and `@fast`.

This part of the code doesn't really care about anything unless it's a tag, so we could accomplish the same thing with `--tags "@ci hello how is it going @fast argh that is not nice"`.

I know that there are complex things possible, e.g. `--tags 'not @foo and (@bar or @zap)'`, according to https://github.com/cucumber/cucumber/tree/master/tag-expressions. I couldn't find docs on this until today!

So, sadly, this PR doesn't implement complex tag expressions!